code/game/objects/_items.dm

```diff
@@ -98,6 +98,12 @@
 	//Items are not dense typically
 	can_block_movement = FALSE
 
+	//Tooltip vars
+	///string form of an item's force. Edit this var only to set a custom force string
+	var/force_string
+	var/last_force_string_check = 0
```

Comment:
use an extension for managing cooldowns and last checks if such is necessary

Reply:
> use an extension for managing cooldowns and last checks if such is necessary

Reverted